src/examples/ex5f.f

```diff
@@ -178,9 +206,15 @@ program ex5f
          endif
 
 !        Set the values for row i
+#if defined(HYPRE_USING_CUDA)
+         tmpi(1) = nnz-1;
```

Comment:
Sorry for commenting out of the blue. Just a nit from a Fortran and *hypre* user that is very interested in this PR! :)
```suggestion
         tmpi(1) = nnz-1
```

Reply:
Thank you, @p-costa! We probably will need your input on how it works.